reconcile/utils/qontract_server_client.py

```diff
@@ -0,0 +1,95 @@
+from urllib.parse import urlparse
+
+import requests
+
+
+class QontractServerClient:
+    """REST API for Qontract Server """
+
+    def __init__(self, url, token=None,
+                 use_sessions=False,
```

Comment:
the `use_sessions` argument doesn't tell much & is exposing internal implementation... I'd suggest calling it `sticky_session` instead.